server/routes.go

```diff
@@ -613,7 +614,9 @@ func (s *Server) CreateModelHandler(c *gin.Context) {
 		defer cancel()
 
 		quantization := cmp.Or(r.Quantize, r.Quantization)
-		if err := CreateModel(ctx, name, filepath.Dir(r.Path), strings.ToUpper(quantization), f, fn); err != nil {
+		if err := CreateModel(ctx, name, filepath.Dir(r.Path), strings.ToUpper(quantization), f, fn); errors.Is(err, errBadTemplate) {
```

Comment:
I would make this a condition in `err != nil` for clarity:

```
if err := CreateModel(ctx, name, filepath.Dir(r.Path), strings.ToUpper(quantization), f, fn); err != nil {
  if errors.Is(err, errBadTemplate) {
    ch <- gin.H{"error": err.Error(), "status": http.StatusBadRequest}
  }
  ch <- gin.H{"error": err.Error()}
}
```

Reply:
this is wrong because now you're sending two errors to the channel